Assets/MixedRealityToolkit/Providers/BaseInputDeviceManager.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.MixedReality.Toolkit.Utilities;
 using System.Collections.Generic;
+using Unity.Profiling;
```

Comment:
Remove?

Reply:
Why remove? This is used for the RequestPointersPerfMarker. The markers are conditionally removed for release builds
"They are conditionally compiled away and thus have zero overhead in non-Developmenet (Release) builds."